app/utils/lib/RoomTypeConfig.js

```diff
@@ -264,4 +264,17 @@ export class RoomTypeConfig {
 		return false;
 	}
 
+	/**
+   * Get receipt's extra token for livechat users
+   * @param {object} message
+   * @return {object} Token for user
+   */
+	getReadReceiptsExtraData(message) {
+		if (!Meteor.isServer) {
```

Comment:
Here the return will be always `return {}`;

Reply:
updated

Reply:
@knrt10, are you sure? The code has not change to me.